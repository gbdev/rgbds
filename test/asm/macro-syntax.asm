	MACRO new ; comment
		println "in with the ", \1
	ENDM ; comment

	new 2

	old: MACRO ; comment
		println "out with the ", \1
	ENDM ; comment

	old 1

