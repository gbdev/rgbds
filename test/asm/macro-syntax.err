warning: macro-syntax.asm(2): [-Wobsolete]
    `old: MACRO` is deprecated; use `MACRO old`
error: macro-syntax.asm(13):
    syntax error, unexpected identifier, expecting newline
error: macro-syntax.asm(15):
    syntax error, unexpected ENDM
error: Assembly aborted (2 errors)!
