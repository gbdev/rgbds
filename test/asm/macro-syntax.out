in with the $2
out with the 
